src/microbe_stage/organelle_components/MovementComponent.cs

```diff
@@ -21,6 +21,36 @@ public MovementComponent(float momentum, float torque)
         Torque = torque;
     }
 
+    public static Vector3 GetForceVector(int orientation)
+    {
+        if (orientation == 0)
```

Comment:
I'd use a switch here, and also add exception throwing if orientation is not one of these valid values

Reply:
```cs
return orientation switch
{
    0 => Constants.VectorDown,
    1 => Constants.VectorDownLeft,
    2 => Constants.VectorUpLeft,
    3 => Constants.VectorUp,
    4 => Constants.VectorUpRight,
    5 => Constants.VectorDownRight,
    _ => throw new NotSupportedException($"invalid orientation {orientation}. Should be 0-5"),
};
```